pyproject.toml

```diff
@@ -42,6 +42,7 @@ standard_library = ["typing_extensions"]
 [tool.ruff]
 target-version = "py38"
 line-length = 120
+src = ["caffe2", "torch", "torchgen", "functorch", "test"]
```

Comment:
Are we sure covered everything? Ruff wont' cover setup.py, benchmarks, github.ci or a few other places it might cover now...

Reply:
This only resolves for directories for first/third-party imports. See <https://docs.astral.sh/ruff/settings/#src>